examples/electricity-trade/cleanup.sh

```diff
@@ -0,0 +1,28 @@
+docker stop sawtooth-shell-default
+docker stop sawtooth-settings-tp-default
+docker stop sawtooth-xo-tp-python-default
+docker stop sawtooth-devmode-engine-rust-default
+docker stop sawtooth-rest-api-default
+docker stop sawtooth-intkey-tp-python-default
+docker stop sawtooth-validator-default
+docker stop geth1
+docker stop gethtestnet_init-chain-geth1_run_1
+docker stop gethtestnet_init-chain-geth1_run_2
+
+docker rm sawtooth-shell-default
+docker rm sawtooth-settings-tp-default
+docker rm sawtooth-xo-tp-python-default
+docker rm sawtooth-devmode-engine-rust-default
+docker rm sawtooth-rest-api-default
+docker rm sawtooth-intkey-tp-python-default
+docker rm sawtooth-validator-default
+docker rm geth1
+docker rm gethtestnet_init-chain-geth1_run_1
+docker rm gethtestnet_init-chain-geth1_run_2
+
+sudo rm -R ../../tools/docker/geth-testnet/data-geth1/geth
+sudo rm ../../tools/docker/geth-testnet/data-geth1/geth.ipc*
```

Comment:
This is only a comment (not a change request) that I'm wondering that the error message "rm: cannot remove '../../tools/docker/geth-testnet/data-geth1/geth.ipc*': No such file or directory" happens (but the file ../../tools/docker/geth-testnet/data-geth1/geth.ipc is removed by this script)
I will accept this change once it is harmless.  If an improvement is found, this part may be fixed on later pull-requests.